frontend/src/lib/lemon-ui/Link/Link.tsx

```diff
@@ -96,13 +100,14 @@ export const Link: React.FC<LinkProps & React.RefAttributes<HTMLElement>> = Reac
         }
 
         return to ? (
+            // eslint-disable-next-line react/forbid-elements
             <a
                 ref={ref as any}
                 className={clsx('Link', className)}
                 onClick={onClick}
                 href={typeof to === 'string' ? to : '#'}
                 target={target}
-                rel={target === '_blank' ? 'noopener noreferrer' : undefined}
+                rel={rel ?? target === '_blank' ? 'noopener noreferrer' : undefined}
```

Comment:
This was purposeful that we didn't allow modifying this, as almost every case would want both of these values. I couldn't find a case where the given ref wasn't essentially achieving the same thing, so maybe we could remove it from the implementers, rather than letting it be overrideable?

Reply:
I wasn't sure about adding it but noticed that anywhere we used utm params we only included `noopener` and not `noreferrer`. Two examples in [this file](https://github.com/PostHog/posthog/pull/18227/files#diff-27ef8d3b192274dc9f18ce6666dae1528fe94bca091a1f500bc13ad24a80aff9). Should I just change the behaviour in those cases?

Reply:
[Checked in](https://posthog.slack.com/archives/C01FHN8DNN6/p1698682639766249) with @andyvan-ph on the SEO side of things and found that we don't want to be adding `noreferrer` to links back to PostHog domains.

Rather than exposing the `rel` attribute I opted to [handle PostHog links within the component](https://github.com/PostHog/posthog/pull/18227/files#diff-8c458d1a7bf59eafea70694283c5a283c6b176f49979fc791aec411b3d594b6eR102)

Reply:
This is a great solution